libcontainerd/process_windows.go

```diff
@@ -42,11 +42,11 @@ func createStdInCloser(pipe io.WriteCloser, process hcsshim.Process) io.WriteClo
 		err := process.CloseStdin()
 		if err != nil && !hcsshim.IsNotExist(err) {
 			// This error will occur if the compute system is currently shutting down
-			if perr, ok := err.(*hcsshim.ProcessError); ok && perr.Err != hcsshim.ErrVmcomputeOperationInvalidState {
+			if perr, ok := err.(*hcsshim.ProcessError); ok && perr.Err != hcsshim.ErrVmcomputeOperationInvalidState && perr.Err != hcsshim.ErrInvalidHandle {
 				return err
 			}
 		}
 
-		return err
+		return nil
```

Comment:
erm, looking again; `hcsshim.IsNotExist()` errors don't need to be returned, correct?